test/sql/include/ts_merge_load.sql

```diff
@@ -0,0 +1,64 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+CREATE USER regress_merge_privs;
```

Comment:
Hm, I am not sure it is safe to create users inside tests this way since users are shared between databases. For instance, will you be able to run same test one after another if same user already exists?

Reply:
We do drop the users in ts_merge_query.sql